### ISSUE TITLE
OP Much?

### ISSUE THREAD

**Issue by DeadlyDad** [ComputerComa/factorio-ore-washing#5]
(Doing a K2 run.)

While I LOVE the idea, it is OP by an order of magnitude, considering I had to use a loader mod because a single steel furnace fed T2 washed ore is pumping out almost 15 iron plates per second, and 4.55 steel furnaces/36.7 T1 miners will create two compressed red belts. 

OTOH, if the throughput was reduced by 90% for most of the process and 60% for miners, 45.5 steel furnaces/110.1 T1 miners to produce 60 iron plates per second would still be a HUGE improvement over the 96 steel furnaces/240 T1 miners that are normally required.

(It's almost 1 in the morning after a long day, so my brain may not be mathing properly. :shrug: )

Cheers!

**Comment by ComputerComa** [comment]
Yeah.. I definately need to rebalance. just havne't gotten around to it. Might look into rebalancing over the weekend.
